project/backend/database/models.py

```diff
@@ -57,26 +58,41 @@ class Admin(BasicUser):
     def __str__(self):
         return self.user.first_name + " " + self.user.last_name
     
+class EnumRequest(enum.Enum):
+    WAITING  = 1
+    APPROVED = 2
+    DENIED   = 3  
+    
+    @classmethod
+    def choices(cls):
+        return [(key.value, key.name) for key in cls]
+    
 class Request(models.Model):
```

Comment:
I have implemented this model in the #426 without enumrequest class. though, the same functionality implemented in the way that [django doc suggests](https://docs.djangoproject.com/en/4.2/ref/models/fields/#choices). I also implemented accept and reject methods too.